tests/system/test/MEZOTransfers.test.ts

```diff
@@ -94,6 +94,68 @@ describe("MEZOTransfers", function () {
     });
   });
 
+  describe("multipleApproveAreHandledProperlyByTheCore", function () {
+    let receipt: any;
+    let tx: any;
+    let receipient = ethers.Wallet.createRandom().address;
+
+    before(async function () {
+      await fixture();
+    });
```

Comment:
Let's put a comment just above the `it`s explaining all state transitions in one place and explicitly pointing that the state is maintained between the `it`s.

Reply:
addressed in 1bba9e8a0b8d6a109a561f3f583958318a1c0ff7